src/winetricks

```diff
@@ -2951,13 +2951,16 @@ _EOF_
     unset W_file W_font
 }
 
+# Note: we use UTF-16 (little endian) in .reg file for native (non-English) font names.
 w_register_font_replacement()
 {
     _W_alias=$1
     shift
     _W_font=$1
+    # UTF-16 BOM (U+FEFF, "0xEF 0xBB 0xBF" in UTF-8)
+    printf "\357\273\277" | iconv -f utf8 -t utf16le > "$W_TMP"/_register-font-replacements.reg
```

Comment:
It looks like BSD doesn't support utf8/utf16le; it uses UTF-8/UTF-16LE (not sure what mac does, but that should be verified as well).

Reply:
Fixed. This works on macOS:
![iconv_test_macos](https://user-images.githubusercontent.com/10556453/72213334-7ec89b80-3530-11ea-83c0-81dd40c18b34.png)